dev-test/githunt/flow.flow.js

```diff
@@ -1,5 +1,7 @@
 /* @flow */
 
+// @flow
```

Comment:
Is it really need? We have this in the 1st line, no? (as `/* @flow */`)

Reply:
yes - if you are using flow-operations plugin alone (as is with near-operation preset)